lib/VM/JSLib/RegExp.cpp

```diff
@@ -595,6 +701,10 @@ CallResult<Handle<JSArray>> directRegExpExec(
   auto inputSHV = SmallHermesValue::encodeStringValue(*S, runtime);
   JSObject::setNamedSlotValueUnsafe(*A, runtime, inputDesc, inputSHV);
 
+  // If R contains any GroupName, then let hasGroups be true.
+  auto groupNames = regexp->getGroupNameMappings(runtime);
```

Comment:
You can write out the type here, it's not obvious from the name.

`Handle<JSObject> groupNames = regexp->getGroupNameMappings(runtime);`